setup.py

```diff
@@ -107,6 +117,53 @@ def build_extensions(self):
 cmdclass['test'] = NoopTestCommand
 cmdclass['build_ext'] = BuildExtraLibraries
 
+
+def _download_jquery_to(dest):
+    # Note: When bumping the jquery-ui version, also update the versions in
+    # single_figure.html and all_figures.html.
+    url = "https://jqueryui.com/resources/download/jquery-ui-1.12.1.zip"
+    sha = 'f8233674366ab36b2c34c577ec77a3d70cac75d2e387d8587f3836345c0f624d'
+    if not os.path.exists(os.path.join(dest, "jquery-ui-1.12.1")):
+        try:
+            os.makedirs(dest)
+        except OSError:
+            pass
+        print("DOWNLOADING JQUERY TO {}".format(dest))
```

Comment:
this print should be moved to download_or_cache (and modified as it's also used for freetype), or just deleted